Neos.ContentRepositoryRegistry/composer.json

```diff
@@ -20,87 +20,5 @@
-            "TYPO3.TYPO3CR-130523180140",
-            "TYPO3.Neos.NodeTypes-201309111655",
-            "TYPO3.Neos-201310021548",
-            "TYPO3.Flow-201310031523",
-            "TYPO3.Flow-201405111147",
-            "TYPO3.Neos-201409071922",
-            "TYPO3.TYPO3CR-140911160326",
-            "TYPO3.Neos-201410010000",
-            "TYPO3.TYPO3CR-141101082142",
-            "TYPO3.Neos-20141113115300",
-            "TYPO3.Fluid-20141121091700",
-            "TYPO3.Neos-20141218134700",
-            "TYPO3.Fluid-20150214130800",
-            "TYPO3.Neos-201407061038",
-            "TYPO3.Flow-201212051340",
-            "TYPO3.Fluid-20141113120800",
-            "TYPO3.Flow-20141113121400",
-            "Typo3.Neos.NodeTypes-201309111655",
-            "TYPO3.TYPO3CR-20140911160326",
-            "TYPO3.Flow-201209251426",
-            "TYPO3.FLOW3-201209201112",
-            "TYPO3.Neos-20150303231600",
-            "TYPO3.Flow-20151113161300",
-            "TYPO3.Form-20160601101500",
-            "TYPO3.Flow-20161115140400",
-            "TYPO3.Flow-20161115140430",
-            "Neos.Flow-20161124204700",
-            "Neos.Flow-20161124204701",
-            "Neos.Twitter.Bootstrap-20161124204912",
-            "Neos.Form-20161124205254",
-            "Neos.Flow-20161124224015",
-            "Neos.Party-20161124225257",
-            "Neos.Eel-20161124230101",
-            "Neos.Kickstart-20161124230102",
-            "Neos.Setup-20161124230842",
-            "Neos.Imagine-20161124231742",
-            "Neos.Media-20161124233100",
-            "Neos.NodeTypes-20161125002300",
-            "Neos.SiteKickstarter-20161125002311",
-            "Neos.Neos-20161125002322",
-            "Neos.ContentRepository-20161125012000",
-            "Neos.Fusion-20161125013710",
-            "Neos.Setup-20161125014759",
-            "Neos.SiteKickstarter-20161125095901",
-            "Neos.Fusion-20161125104701",
-            "Neos.NodeTypes-20161125104800",
-            "Neos.Neos-20161125104802",
-            "Neos.Kickstarter-20161125110814",
-            "Neos.Flow-20161125124112",
-            "Neos.Neos-20161125122412",
-            "TYPO3.FluidAdaptor-20161130112935",
-            "Neos.Fusion-20161201202543",
-            "Neos.Neos-20161201222211",
-            "Neos.Fusion-20161202215034",
-            "Neos.ContentRepository-20161219093512",
-            "Neos.Fusion-20170120013047",
-            "Neos.Flow-20180415105700",
-            "Neos.Fusion-20161219092345",
-            "Neos.Media-20161219094126",
-            "Neos.Neos-20161219094403",
-            "Neos.Neos-20161219122512",
-            "Neos.Fusion-20161219130100",
-            "Neos.Neos-20161220163741",
-            "Neos.Neos-20170115114620",
-            "Neos.Flow-20170125103800",
-            "Neos.Seo-20170127154600",
-            "Neos.Flow-20170127183102",
-            "Neos.Fusion-20180211175500",
-            "Neos.Fusion-20180211184832"
-        ]
```

Comment:
Not related to the issue, but I don't think that we need to keep track of applied core migrations in core packages. It bloats the composer file without real benefits